etc/kayobe/kolla/config/prometheus/ceph.rules

```diff
@@ -154,7 +154,7 @@ groups:
 
   # alert on nic packet errors and drops rates > 1 packet/s
   - alert: NetworkPacketsDropped
-    expr: irate(node_network_receive_drop_total{device=~"en.*|eth.*"}[5m]) + irate(node_network_transmit_drop_total{device=~"en.*|eth.*"}[5m]) > 1
+    expr: irate(node_network_receive_drop_total{device!~"lo|br.*|.*-ovs|tap.*"}[5m]) + irate(node_network_transmit_drop_total{device!~"lo|br.*|.*-ovs|tap.*"}[5m]) > 1
```

Comment:
This will catch, among others:
```
bond.*
docker0
genev_sys_6081
ovs-system
p-brbond.*
qbr.*
qvb.*
vxlan_sys_4789
```
The previous regex has already been filtering out OVS bridges - I thought we were only expanding it with `|bond.*`.

Does by the way `|ib.*` make sense too?